content/03-how-tos/03-how-to-configure-mantis.mdx

```diff
@@ -0,0 +1,146 @@
+```
+include "app.conf"
+
+mantis {
+  blockchains {
+    network = "etc"
+  }
+
+  network {
+
+    server-address {
+      # Listening interface for Ethereum protocol connections
+      interface = "0.0.0.0"
+
+      # Listening port for Ethereum protocol connections
+      port = 9076
+    }
+  }
+}
+```
+
+5. Start Mantis with this command:
+```
+cd ~/mantis-3.0
+./mantis-launcher [your name]
+``` 
+
+## Using the Dkey
+To change configuration options at startup on the command line, you can use Dkey arguments to define the key and value as follows:
+
+```
+./mantis-launcher [config name] -D[key name]=[value]
+```
+
+Replace `[config-name]` with your configuration file name, [key name] with the name of the key, and [value] with the new value.
+
+Prefix the key name with all the higher-level key names as defined in base.conf, like this:  
+`-Dmantis.network.port='9076'`
+
+# Additional notes
+## Data Directory
+By default, Mantis stores the blockchain data, logs, keystore, and other files it creates in the user’s home folder under a subfolder called `.mantis`. Each network has a separate subfolder, that is `/.mantis/etc` for the Ethereum Classic mainnet, `/.mantis/mordor` for the Mordor testnet, or `/.mantis/sagano` for the Sagano testnet.
+
+In the configuration files, you can add or change a mantis.datadir option as follows:
+
+```
+mantis {
+  # Base directory where all the data used by the node is stored, including blockchain data and private keys
+   datadir = ${user.home}"/.mantis/"${mantis.blockchains.network}
+}
+```
+You can also use a Dkey parameter on the command line:
+
+```
+-Dmantis.datadir=/new/datadir/location
+```
+
+> Note: You cannot use the `~` shortcut to define the new location.
+
+## Discovery
+The discovery function is active by default. Mantis queries the network for additional potential nodes using a partial implementation of the Kademlia discovery protocol.
+
+### Configuration file
+To prevent this and use *only* the bootstrap nodes as listed in the specific chain configuration, find the `discovery-enabled` key and set value to `false` to disable the discovery functionality:
+`discovery-enabled = false`.
+
+### Dkey
+```
+-Dmantis.network.discovery.discovery-enabled=true
+```
+
+## Fast sync
+`Fast sync` can synchronize your local blockchain data faster than a full (or archival) sync. Mantis downloads all blocks and only verifies the proof of work of each block instead of downloading and reprocessing every transaction from genesis.
+
+To disable this, you can:
+
+### Configuration file
+Edit the configuration file by changing the value of the following property: `do-fast-sync` to false
+`do-fast-sync = false`
+
+### Dkey
+```
+-Dmantis.sync.do-fast-sync=false # to run without fast-sync
+```
+
+## Pruning
+Pruning is the process of deciding which blocks are not part of the main chain and not storing them.   
+You can control this with these three values of the pruning key:  
```

Comment:
The config key here is `mantis.pruning.mode` (as in the example below) and the default value is `"basic"`.